### ISSUE TITLE
Application - Cloud Host

### ISSUE THREAD

**Issue by hebi0415** [deepflowys/deepflow#1394]
### Feature request

grafana中的模板监控中，client error 中Protocol和Enum(l7_protocol)的报错都执行DNS，无法看到准确的报错码

**Comment by mytting** [comment]
@liqian1834 The application request log is added under the dashboard

**Comment by Shuimo03** [comment]
请问这个issues解决了吗？如果没有的话，我可以来试下嘛？

**Comment by mytting** [comment]
> 请问这个issues解决了吗？如果没有的话，我可以来试下嘛？

Of course, looking forward to your PR. We originally planned to add DNS request logs in the dashboard to display, but we haven't had time to do it recently.

**Comment by Shuimo03** [comment]
Thanks, I'll try it~

**Comment by Shuimo03** [comment]
sorry, some time ago because of my reason, so this issue not resolve, I will be in the these days to solve it, sry
